webserver/views/api/v1/core.py

```diff
@@ -155,6 +155,37 @@ def _parse_bulk_params(params):
     return [x for x in ret if not (x in seen or seen.add(x))]
 
 
+def check_bad_request_for_multiple_recordings():
+    """Checking whether the recording ids throw exceptions
```

Comment:
You should mention exactly what exceptions will be thrown and when. Also, what happens if the data is correct.